internal/pkg/specs/user_email.go

```diff
@@ -0,0 +1,20 @@
+package specs
+
+import (
+	"fmt"
+
+	errors "github.com/joshsoftware/profile_builder_backend_go/internal/pkg/errors"
+)
+
+// UserEmailRequest is the request for sending an email to the user
+type UserSendInvitationRequest struct {
```

Comment:
This struct is really need for single params ? just for profileID ?
can we pass profielD in SendUserInvitation Method signature ?

Reply:
If in future if need to pass any other parameter , by considering this i have passed struct as a parameter